manager/apis/app/v1alpha1/fybrikmodule_types.go

```diff
@@ -40,24 +40,8 @@ const (
 )
 
 // ModuleCapability indicates at a high level what is being performed - often on a datset, but potentially other things as well
-// +kubebuilder:validation:Enum=copy;read;write;transform
 type CapabilityType string
 
-// TODO - Should these come from the taxonomy?
-const (
-	// Copy moves data from one location to another - i.e implicit copy
-	Copy CapabilityType = "copy"
-
-	// Write is accessed from within an application, typically through an SDK
-	Write CapabilityType = "write"
-
-	// Read is accessed from within an application, typically through an SDK
-	Read CapabilityType = "read"
-
-	// Transform processes and changes data
-	Transform CapabilityType = "transform"
-)
-
```

Comment:
Do not remove the constants.

Reply:
Ok I kept the constants and only removed the kubebuilder validation for `CapabilityType`